src/scss/02_atoms/form/_autocomplete.scss

```diff
@@ -0,0 +1,175 @@
+$autocompleteConfig: (
+  paddingRight: rem(60px),
+  list: (
+    hoverBackgroundColor: color(background, dark),
+    maxOptionsToShow: 4,
+    paddingLeft: rem(15px),
+    paddingRight: rem(15px),
+    selectedBackgroundColor: color(brand-alt, light),
+    selectedColor: color(brand-alt, dark),
+  ),
+  searchIcon: (
+    backgroundColor: color(shape),
+  ),
+  searchReset: (
+    backgroundColor: color(shape, light),
+    borderRadius: 50%,
+    color: color(shape, dark),
+    offset: rem(10px),
+    size: rem(30px),
+  ),
+  noResult: (
+    color: color(text, dark),
+    fontSize: size(xsmall),
+  )
+) !default;
+
+.form-autocomplete {
+  position: relative;
+
+  @include mqDown(small) {
+    flex: 1;
+  }
+
+  @include formFieldSizes;
+}
+
+.form-autocomplete__input {
+  padding-right: get($autocompleteConfig, paddingRight);
+
+  .is-open & {
```

Comment:
manca il selettore del padre. Occhio che questo è come dire: 
`*.is-open .form-autocomplete__input`